tools/site_admin/show_job_log.php

```diff
@@ -50,7 +50,7 @@
 echo "<th>" . _("Event") . "</th>";
 echo "<th>" . _("Comments") . "</th>";
 echo "</tr>";
-foreach (get_job_log_entries($start_timestamp, $filename, $event) as $row) {
+foreach (get_job_log_entries($start_timestamp, $job, $event) as $row) {
```

Comment:
Does the definition of this function still use $filename, could that be confusing?

Reply:
It does because the field in the table itself is "filename". I don't think this will be too confusing.